src/main/java/com/emc/ecs/nfsclient/network/ClientIOHandler.java

```diff
@@ -81,7 +81,9 @@ public InetSocketAddress getRemoteAddress() {
      * org.jboss.netty.channel.ChannelStateEvent)
      */
     public void channelConnected(ChannelHandlerContext ctx, ChannelStateEvent e) throws Exception {
-        LOG.info("Connected to: {}", getRemoteAddress());
+        if (LOG.isDebugEnabled()) {
```

Comment:
the first thing it does is call isDebugEnabled... and since there's no string concatenation (which is quite fast anyway), it would be about the same as if you called isDebugEnabled() yourself